### PR TITLE
Upgrade webpack and set up build now that we're using Flow

### PR DESCRIPTION
**Pull request by jeremywiebe**
## Summary:

Upgrades to latest webpack and configures build (now that we use Flow).

Issue: "none"

## Test plan:

`yarn build` succeeds and doesn't "dirty" build/kmath.js

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,19 +1,28 @@
-const path = require('path')
+const path = require("path");
```
Comment (jaredly) [inline webpack.config.js]: unintentional?

**Review by jaredly — APPROVED** [review]

File: webpack.config.js (hunk 1 above)
Comment (jeremywiebe) [reply webpack.config.js]: `prettier` decided this.  Maybe I should exclude this file from Prettier?

Comment (jeremywiebe) [reply webpack.config.js]: I'll revert for now... no need for extra churn.

File: build/kmath.js [hunk 2]
```diff
@@ -1,628 +1 @@
-(function webpackUniversalModuleDefinition(root, factory) {
```
Comment (jeremywiebe) [inline build/kmath.js:1]: FWIW, this is now a minified file, which I think is what we want for a build file.

**Pull request merged**